ailab/db/finesse/__init__.py

```diff
@@ -14,14 +14,14 @@ def load_prompt(prompt_path, filename):
     return content
 
 
-def load_json_template(json_path=DEFAULT_JSON_PATH):
+def load_json_template(filename, json_path=DEFAULT_JSON_PATH):
     """
     Load a JSON template from the specified path.
 
     Returns:
     str: A JSON-formatted string representing the loaded data.
     """
-    json_file_path = json_path + "/json_template.json"
+    json_file_path = json_path + "/" + filename + ".json"
```

Comment:
better to use os.path.join and other standardized path manipulation functions.

https://docs.python.org/3/library/os.path.html#os.path.join